tests/test_selenium_brands_page.py

```diff
@@ -0,0 +1,59 @@
+from page_objects.home_page import HomePage
+import pytest
+
+
+@pytest.mark.smoke
+def test_open_brands_page(create_driver):
+    """
+    open site->open brands page by footer menu
+    """
+    driver = create_driver
+    brands_page = HomePage(driver).scroll_click_on_brands()
```

Comment:
same here/ code duplication and verification only is_display 
fix all around file